datasets/gov/hi/vaistu_suvartojimas/antimikrobinis.csv

```diff
@@ -0,0 +1,27 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/hi/vaistu_suvartojimas,,,,,,,,,,,,Antimikrobinių vaistinių preparatų suvartojimo Lietuvoje duomenys,
+,,,,,,,,,,,,,,
+,,,,Antimikrobinis,,,,,,3,,,Antimikrobinių vaistinių preparatų suvartojimas,
+,,,,,metai,integer,yr,,,2,open,,Antimikrobinių vaistinių preparatų suvartojimo statistikos vedimo metai,
+,,,,,istaiga,ref,/datasets/gov/vaspvt/licencijos/Istaiga[ja_kodas],,,4,open,,Sveikatos priežiūros įstaigos JAR kodas,
+,,,,,ligonine,string,,,,2,open,,Ligoninės pavadinimas,
+,,,,,tipas,string,,,,4,open,,Ligoninės tipas,"Ligoninių tipai: bendrojo pobūdžio ligoninė, palaikomojo gydymo ir slaugos ligoninė, specializuota ligoninė"
+,,,,,atc,string,,,,4,open,,Tarptautinę anatominio terapinio cheminio indekso klasifikacijos kodas,
+,,,,,veiklioji,string,,,,4,open,,Vaistinės veikliosios medžiagos pavadinimas,
+,,,,,komercinis,string,,,,4,open,,Vaistinio preparato komercinis pavadinimas,
+,,,,,pakid,string,,,,4,open,,Antimikrobinio vaistinio preparato pakuotės identifikavimo kodas,"Reikšmę turi tik tiek preparatai, kurie yra registruoti Europos vaistų agentūros"
+,,,,,stiprumas,string,,,,4,open,,Antimikrobinio vaistinio preparato pakuotės stiprumo reikšmė(-s),

```

Comment:
Norėtųsi skaidyti į du atskirus stulpelius "Stiprumas" ir "Stiprumo vienetas", o vnt ir gramas kiek suprantu čia kiekis gramais to preparato. Tai šitam mažinčiau brandos lygį iki 2.